src/components/AoDropdownLink.vue

```diff
@@ -0,0 +1,41 @@
+<template functional>
+  <div
+    :class="{ disabled: props.disabled}"
+    class="ao-dropdown__link"
+  >
+    <slot />
```

Comment:
can we make this a text prop instead? we want these to only be text right?

Reply:
Could want to pass in a span with an href to link to somewhere else so I would rather have this be open.

Reply:
sounds good